src/toc_tree.tsx

```diff
@@ -55,6 +56,16 @@ interface IState {}
  * @private
  */
 class TOCTree extends React.Component<IProperties, IState> {
+  /**
+   * Returns a component which renders a table of contents tree.
+   *
+   * @param props - component properties
+   * @returns component
+   */
+  constructor(props: IProperties) {
+    super(props);
+  }
+
```

Comment:
No, it is not necessary. I usually include it as a matter of course so that the `constructor` is documented and setup in the event that component is necessary (e.g., as in `toc.tsx`). I can remove, if this is a blocker.